src/test/components/MenuButtons.test.js

```diff
@@ -138,22 +138,23 @@ describe('app/MenuButtons', function() {
       // the implementation logic of component and test the things user can see
       // and interract with `waitForElement`. But since sharing is async and we
       // can't catch other visual DOM changes here, we had to do this.
-      await waitForDomChange({ container: getMenuButtonsContainer() }).then(
-        mutationsList => {
-          const mutation = mutationsList[0];
-          expect(
-            mutation.target.classList.contains('currentButtonIsPermalinkButton')
-          ).toEqual(true);
-          expect(
-            mutation.target.classList.contains('currentButtonIsShareButton')
-          ).toEqual(false);
-          expect(
-            mutation.target.classList.contains(
-              'currentButtonIsSecondaryShareButton'
-            )
-          ).toEqual(true);
-        }
-      );
+      await waitForDomChange({
+        container: getMenuButtonsContainer(),
+        mutationObserverOptions: { attributes: true },
```

Comment:
Latest jsdom supports MutationObservers -- it looks like the polyfill had a bug here and we relied on it...